packages/api/src/client/types/app/bsky/graph/assertCreator.ts

```diff
@@ -0,0 +1,4 @@
+/**
+* GENERATED CODE - DO NOT MODIFY
+*/
+export const MAIN = 'app.bsky.graph.assertCreator#main'
```

Comment:
How/where do we imagine this constant being used?  It would be sort of nice if the string values used for the tokens appeared as a const somewhere, too!

General thought bouncing around my brain is that some deeper connection between the existence of a token and the string value we use to represent them might help crystallize lexicon's concept of tokens a bit better.  Also recognize I may just not have the right feel for them yet.

Reply:
That's sort of a wonky side-effect of what I was mainly thinking about, which is non-main tokens having a const exported. But the main usage of tokens in our codegen is in the main index.ts, where they're enumerated in dicts.

Open to ideas for better approaches to tokens. The trick is, they still need to ultimately act as strings, and we can't do a closed set so enums are a bit awkward to use.